### PR TITLE
translations updated issue #137

### PR DESCRIPTION
**Pull request by cardawid**
translations are up-to-date now

### CODE REVIEW

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -5,16 +5,17 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: piplmesh\n"
-"Report-Msgid-Bugs-To: https://github.com/wlanslovenija/PiplMesh\n"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po:8]: No, don't change that.

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -36,70 +37,80 @@ msgstr "ženska"
...
 msgstr "Spol"
 
-#: .\account\forms.py:128
+#: .\account\forms.py:101
 msgid "Birth date"
 msgstr "Datum rojstva"
 
+#: .\account\models.py:32
+msgid "username"
+msgstr "Uporabniško ime"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: Don't change case.

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -160,21 +181,64 @@ msgstr "Spremeni geslo"
...
+"be enabled."
+msgstr ""
+
+#: .\frontend\templates\404.html.py:5 .\frontend\templates\404.html.py:10
+msgid "404 Not Found"
+msgstr ""
+
+#: .\frontend\templates\404.html.py:13
+msgid "The page you requested cannot be found."
+msgstr "Stran, ki ste jo zahtevali ne obstaja."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: Comma after _zahtevali_.

Comment (cardawid) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: Fixed. Everything.

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -160,21 +181,64 @@ msgstr "Spremeni geslo"
...
+msgid "User image"
+msgstr "Slika uporabnika"
+
+#: .\frontend\debug.py:6
+msgid "File"
+msgstr "Datoteka"
+
+#: .\frontend\templates\403.html.py:5 .\frontend\templates\403.html.py:10
+msgid "403 Forbidden"
+msgstr "403 Prepovedana stran"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _403 Prepovedano_?

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -194,38 +271,54 @@ msgstr "Išči"
...
 msgstr "Rezultati za:"
 
-#: .\frontend\templates\form\form.html.py:10
+#: .\frontend\templates\userlist.html.py:4
+msgid "Show/Hide Users"
+msgstr "Prikaži/Skrij uporabnika"
+
+#: .\frontend\templates\userlist.html.py:4
+msgid "Online Users"
+msgstr "Dosegljivi uporabniki"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Povezani uporabniki_?

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -194,38 +271,54 @@ msgstr "Išči"
 msgid "Results for:"
 msgstr "Rezultati za:"
 
-#: .\frontend\templates\form\form.html.py:10
+#: .\frontend\templates\userlist.html.py:4
+msgid "Show/Hide Users"
+msgstr "Prikaži/Skrij uporabnika"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _uporabnikE_

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -182,10 +246,23 @@ msgstr "Slika uporabnika"
...
+"You are accessing the site outside of the <i>wlan slovenija</i> open "
+"wireless network. Because of this your current location cannot be determined "
+"automatically and we have to determine it through your browser. By providing "
+"us with your location we will be able to deliver you relevant information "
+"about your surroundings."
+msgstr ""
+
+#: .\frontend\templates\outside.html.py:11
+msgid "Send my current location"
+msgstr "Pošlji trenutno lokacijo"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _mojo_

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -160,21 +181,64 @@ msgstr "Spremeni geslo"
...
+"be enabled."
+msgstr ""
+
+#: .\frontend\templates\404.html.py:5 .\frontend\templates\404.html.py:10
+msgid "404 Not Found"
+msgstr "404 Datoteke ni mogoče najti"
+
+#: .\frontend\templates\404.html.py:13
+msgid "The page you requested cannot be found."
+msgstr "Stran, ki ste jo zahtevali, ne obstaja."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Željena stran ne obstaja._

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -160,21 +181,64 @@ msgstr "Spremeni geslo"
...
+
+#: .\frontend\templates\403.html.py:18
+msgid ""
+"For accessing this site and using all features cookies in your browser must "
+"be enabled."
+msgstr ""
+
+#: .\frontend\templates\404.html.py:5 .\frontend\templates\404.html.py:10
+msgid "404 Not Found"
+msgstr "404 Datoteke ni mogoče najti"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _404 Ne obstaja_

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -160,21 +181,64 @@ msgstr "Spremeni geslo"
...
+msgid "File"
+msgstr "Datoteka"
+
+#: .\frontend\templates\403.html.py:5 .\frontend\templates\403.html.py:10
+msgid "403 Forbidden"
+msgstr "403 Prepovedana stran"
+
+#: .\frontend\templates\403.html.py:13
+msgid "You don't have permission to access the page you requested."
+msgstr "Nimate pravic za dostop do željene strani."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Nimaš_?

**Pull request merged**